packages/migrate/test/__snapshots__/migrate.test.ts.snap

```diff
@@ -245,6 +245,28 @@ export default class Hello extends Component<HelloSignature> {
 "
 `;
 
+exports[`fix > .gts > with more errors 1`] = `
+"// @ts-expect-error @rehearsal TODO TS2345: Argument of type 'typeof MyForm' is not assignable to parameter of type 'abstract new () => HasContext<AnyContext'.
+export default class MyForm {
+  submit(e: Event) {
+    const something = {
+      run:  () => {
+        console.log(this);
+      }.bind(this),
```

Comment:
This is invalid syntax.
https://www.typescriptlang.org/play?#code/MYGwhgzhAECyCeBhA9gWwA7IHYFMsBcAoAb2kOggFcAjVAS3wAocAuaAUQDc98BKaYuWjDg2CPgpoc+ABZ0sAc2gBeAUOHCATpSxtojfsoB8ajWeiisEZCBwA6EMgWNZdCLwDc6jQF871eQATFzl3ABpvHy8zTWlKTSxJVGk5RWjoH0JMoA
<img width="462" alt="Screenshot 2023-07-26 at 4 16 30 PM" src="https://github.com/rehearsal-js/rehearsal-js/assets/129781/165b9d70-67ba-4c25-ada4-16f39d8ccac6">

Reply:
WIP. I allowed this codefix to be applied just for regex test during development. Will be disabled soon.

Reply:
Disabled now.